src/silx/gui/data/DataViewer.py

```diff
@@ -349,27 +349,45 @@ def getViewFromModeId(self, modeId):
         :param int modeId: Requested mode id
         :rtype: silx.gui.data.DataViews.DataView
         """
+        if modeId == STACK_MODE:
+            deprecated_warning(
+                "Argument",
+                "DataViews.STACK_MODE",
+                replacement="DataViews.IMAGE_MODE",
+                since_version="3.0.0",
+            )
+            return None
```

Comment:
I prefered to follow the docstring instead of setting `modeId` to `IMAGE_MODE`: `Return None if modeId does not correspond to an existing view`